docs/source/interfaces/indirect/Inelastic Data Analysis.rst

```diff
@@ -23,7 +23,7 @@ The other two tabs in the interface perform transformations on data and are docu
 
 These interfaces do not support GroupWorkspaces as input.
 
-.. interface:: Data Analysis
+.. image:: ../../images/Inelastic_Data_Analysis_GUI.png
   :width: 650
 
```

Comment:
Maybe you want the width and align parameters to be the same on both pages. 650 width I'd say is good as a minimum for this image, I like them large!